react/Checkbox/index.jsx

```diff
@@ -34,7 +37,17 @@ const Checkbox = props => {
         disabled={disabled}
         {...restProps}
       />
-      <span>{label || children}</span>
+      <span>
+        {/* TODO: Change 'check-circleless' to 'check' when icons are cleaned up */}
```

Comment:
It would be nice to have the PR linked here so that if someone passes after, (s)he can click on the link, see that the PR is merged and do the change.